app/controllers/players_controller.rb

```diff
@@ -0,0 +1,87 @@
+class PlayersController < ApplicationController
+  skip_before_action(:force_player_sign_in, { :only => [:new_registration_form, :create] })
+  
+  def new_registration_form
+    render({ :template => "player_sessions/sign_up.html.erb" })
+  end
+
+  def create
+    @player = Player.new
+          
+    @player.email = params.fetch(:email, @player.email)
+                      
+    @player.password = params.fetch(:password, nil)
+    @player.password_confirmation = params.fetch(:password_confirmation, nil)
+                
+    @player.first_name = params.fetch(:first_name, @player.first_name)
+                
+    @player.middle_name = params.fetch(:middle_name, @player.middle_name)
+                
+    @player.bio = params.fetch(:bio, @player.bio)
+                
+    @player.a_bool = params.fetch(:a_bool, @player.a_bool)
```

Comment:
I think this is the only case when there should be a fallback to `fetch` — when it's a `boolean` column and therefore we generate a checkbox.